compositions/upbound-aws-provider/iam-policy/definition.yaml

```diff
@@ -40,6 +44,8 @@ spec:
                       type: string
                     providerConfigName:
                       type: string
+                    region:
```

Comment:
Region was left out of spec because of IAM services spec. I am not inclined to expand it to accommodate Lambda permissions.